src/main/java/com/pjait/byt/pettracker/Pet.java

```diff
@@ -77,7 +80,8 @@ public void setCollar(Collar collar) {
     }
 
     public void setPredefinedZone(Location[] loc) {
-        predefinedZone = loc;
```

Comment:
could have also used @Size(min=2, max=2) from validation-api but this also works for the time being.